public/js/components/Editor.js

```diff
@@ -221,22 +223,30 @@ const Editor = React.createClass({
     }
   },
 
-  render() {
+  renderBreakpoints() {
     const { breakpoints, sourceText } = this.props;
     const isLoading = sourceText && sourceText.get("loading");
 
+    if (isLoading) {
+      return;
+    }
+
+    return breakpoints.valueSeq().map(bp => {
+      return Breakpoint({
+        key: makeLocationId(bp.location),
+        breakpoint: bp,
+        editor: this.editor && this.editor.codeMirror
+      });
+    });
+  },
+
+  render() {
     return (
       dom.div(
         { className: "editor-wrapper devtools-monospace" },
         dom.div({ className: "editor-mount" }),
-        !isLoading &&
-          breakpoints.valueSeq().map(bp => {
-            return Breakpoint({
-              key: makeLocationId(bp.location),
-              breakpoint: bp,
-              editor: this.editor && this.editor.codeMirror
-            });
-          })
+        this.renderBreakpoints(),
```

Comment:
Yeah, I can see where you're coming from. I've seen both styles, personally I'm not a fan of a single large expression and the ternaries, booleans, etc that go into it working. 

Looking at `renderBreakpoints` specifically. The function is a good example of a 15 line function with no dependencies, which simplifies the `render` function. 

I'd be okay keeping more of the logic in `render` if you feel like that would be important for a consistent style.